test/Tests/src/Data/Time/Date_Part_Spec.enso

```diff
@@ -0,0 +1,99 @@
+from Standard.Base import all
+
+import Standard.Test
+
+spec name create_new_date =
```

Comment:
I really like that this spec is separate so that it can be easily re-used by both `Date_Time` and `Date` :heart: